glusterapi/volume.py

```diff
@@ -1,132 +1,263 @@
-from common import BaseAPI
-from exceptions import GlusterApiInvalidInputs
+"""This module contains the python  glusterd2 volume api's implementation."""
+import httplib
+import json
 
+from glusterapi.common import BaseAPI, validate_uuid, validate_volume_name
+from glusterapi.exceptions import GlusterApiInvalidInputs
 
-class TransportType:
+
+class TransportType(object):
     TCP = "tcp"
     RDMA = "rdma"
     Both = "tcp,rdma"
 
+    def check(self, transport_type):
+        if transport_type not in (self.TCP, self.RDMA, self.Both):
+            raise GlusterApiInvalidInputs(
+                "Transport type %s not supported" % transport_type)
+
+
+def validate_brick(bricks=None):
+    """
+    Validate brick pattern.
+
+    :param bricks: (string) in the form of nodeid:brickpath
```

Comment:
(list) in the form of nodeid:brickpath? See my comment below.

Reply:
yes it is in form of nodeid:brick list